ui/src/tasks/actions/index.ts

```diff
@@ -422,7 +422,7 @@ export const saveNewScript = (script: string, preamble: string) => async (
       orgs: {org},
     } = getState()
 
-    await client.tasks.createByOrgID(org.id, fluxScript)
+    await client.tasks.createByOrgID(org.id, fluxScript, null)
```

Comment:
What is the null for here? Could we name this in a const and provide it for clarity?

Reply:
**is there a diff between providing `null` vs excluding it as `undefined`.

Reply:
Palak is working on https://github.com/influxdata/influxdb/issues/14020, which required updating the client so that a token is always passed. But the influxdb side of her work hasn't landed yet and I wanted to update the dependency in the meantime, hence this funkiness. 

Passing `null` just means to preserve the old behavior. Not obvious at all, but it will be shortly removed once Palak merges her work.

Reply:
😬